examples/gpt/schedule.py

```diff
@@ -414,9 +416,11 @@ def order_args_fn(*args, **kwargs):
 
 
 def broadcast_input(sch):
-    def broadcast_input(inputs):
+    group_src_rank = _get_global_rank(sch.group, 0)
+
+    def _broadcast_input(module, inputs):
         for inp in inputs:
```

Comment:
Some of the `inputs` are None, when `--disable_pipeline` is used, and `num_mp = 8`

Reply:
we will leave this one to the pr from you.